packages/react-button/src/components/Button/ButtonBase.tsx

```diff
@@ -1,10 +1,10 @@
 import * as React from 'react';
 import { ButtonProps } from './Button.types';
-import { compose } from '@fluentui/react-compose';
+import { compose, ComposePreparedOptions } from '@fluentui/react-compose';
 import { useButton } from './useButton';
 
 export const ButtonBase = compose<'button', ButtonProps, ButtonProps, {}, {}>(
-  (props, ref, composeOptions) => {
+  (props: ButtonProps, ref: React.Ref<HTMLElement>, composeOptions: ComposePreparedOptions<ButtonProps>) => {
```

Comment:
Should it be `HTMLButtonElement`?

```suggestion
  (props: ButtonProps, ref: React.Ref<HTMLButtonElement>, composeOptions: ComposePreparedOptions<ButtonProps>) => {
```